Cargo.toml

```diff
@@ -10,8 +10,8 @@ async-trait = "0.1"
 clap = { version = "4.4", features = ["derive", "env"] }
 derive_more = "0.99"
 futures = "0.3"
-hotshot-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.14" }
-serde = { version = "1.0", features = ["derive"] }
+hotshot-types = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.20" }
+serde = { version = "1.0.197", features = ["derive"] }
```

Comment:
Not like this. This is a dependabot thing, but it's bad. Just update the Cargo.lock instead.

In this case, you would do `cargo update hotshot-types` after making the change to `Cargo.toml`